docs/src/main/asciidoc/overview.adoc

```diff
@@ -149,6 +149,26 @@ Flag to set the binder health as `down`, when any partitions on the topic, regar
 +
 Default: `false`.
 
+spring.cloud.stream.kafka.binder.truststoreLocationOnFileSystem::
+When truststore location is given as classpath URL (classpath:), then the binder moves the resource from the classpath location inside the JAR file to a location on the filesystem.
+The file will be moved to the location specified as the value for this property.
+It can be specified as the full absolute path of the file to move to.
+The file name can be different from the one inside the classpath if need be in order to avoid naming collisions in case of multiple applications running on the same machine.
+If this value is not set and the truststore is provided as a classpath resource, then it will be moved to System's temp directory as returned by `System.getProperty("java.io.tmpdir")` with the file name preserved as it is in the classpath.
+This is also true, if this value is present but the diretory cannot be found on the filesystem.
++
+Default: none.
+
+spring.cloud.stream.kafka.binder.keystoreLocationOnFileSystem::
+When keystore location is given as classpath URL (classpath:), then the binder moves the resource from the classpath location inside the JAR file to a location on the filesystem.
+The file will be moved to the location specified as the value for this property.
+It can be specified as the full absolute path of the file to move to.
+The file name can be different from the one inside the classpath if need be in order to avoid naming collisions in case of multiple applications running on the same machine.
+If this value is not set and the keystore is provided as a classpath resource, then it will be moved to System's temp directory as returned by `System.getProperty("java.io.tmpdir")` with the file name preserved as it is in the classpath.
+This is also true, if this value is present but cannot be found on the filesystem.
```

Comment:
Use the same wording as the sentence above.

Reply:
That said, I don't see any logic to fallback to the temp dir if the supplied value is not valid.